internal/cache/testdata/TestUpdateFromUserInfo/golden/local_groups_are_filtered

```diff
@@ -1,13 +0,0 @@
-GroupByID:
```

Comment:
just double confirm, this is replaced by local groups gets ignored now, right? (better name btw!) :)

Reply:
Exactly!